Pre_processing.py

```
@@ -0,0 +1,74 @@
+        remover = StopWordsRemover(inputCol="tokens", outputCol="filtered_tokens")
+        df_out = remover.transform(df)
+        return(df_out)
+
+    def hash_df(self,df):
+        #Hash the filtered token column
+        hashing_tf = HashingTF(inputCol="filtered_tokens", outputCol="raw_features")
+        df_out =  hashing_tf.transform(df)
+        return (df_out)
+
+    def idf_df(self,train_df,test_df):
+        #Create a vectoroizer using the train set and and apply it to both train and test set 
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)